x/claims/keeper/ibc_callbacks.go

```diff
@@ -134,51 +134,60 @@ func (k Keeper) OnRecvPacket(
 	}
 
 	recipientClaimsRecord, recipientRecordFound := k.GetClaimsRecord(ctx, recipient)
+	amt, err := ibc.GetTransferAmount(packet)
+	if err != nil {
+		return channeltypes.NewErrorAcknowledgement(err.Error())
+	}
 
-	// Handle four cases of comparing recipient and sender addresses and checking
-	// for claims records
-	switch {
-	case senderRecordFound && recipientRecordFound && !sameAddress:
-		// case 1: both sender and recipient are distinct and have a claims record
-		// -> merge sender's record with the recipient's record and claim actions that
-		// have already been claimed by one or the other
-		recipientClaimsRecord, err = k.MergeClaimsRecords(ctx, recipient, senderClaimsRecord, recipientClaimsRecord, params)
-		if err != nil {
-			return channeltypes.NewErrorAcknowledgement(err.Error())
-		}
-
-		// update the recipient's record with the new merged one and delete the
-		// sender's record
-		k.SetClaimsRecord(ctx, recipient, recipientClaimsRecord)
-		k.DeleteClaimsRecord(ctx, sender)
-		logger.Debug(
-			"merged sender and receiver claims records",
-			"sender", senderBech32,
-			"receiver", recipientBech32,
-			"total-claimable", senderClaimsRecord.InitialClaimableAmount.Add(recipientClaimsRecord.InitialClaimableAmount).String(),
-		)
+	// Cases with SenderRecordFound.
+	// They require a merge or migration of claims records. To prevent this
+	// happening by accident, they are only executed, when the sender transfers
+	// the specified IBCTriggerAmt.
+	if amt.Equal(types.IBCTriggerAmt) {
```

Comment:
Avoid identation
if !amt.Equal(types.IBCTriggerAmt) {
    return ack
}

switch {

Reply:
@ramacarlucho That's what I had first, but this requirement is only for Cases with SenderRecordFound